### PR TITLE
Add last modification date and user

### PR DESCRIPTION
**Pull request by archendel**

### CODE REVIEW

File: src/main/java/org/gridsuite/directory/server/services/ConsumerService.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package org.gridsuite.directory.server.services;
```
Comment (niconoir) [inline src/main/java/org/gridsuite/directory/server/services/ConsumerService.java:1]: Missing header

File: src/main/java/org/gridsuite/directory/server/services/ConsumerService.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import java.util.UUID;
+import java.util.function.Consumer;
+
+import org.gridsuite.directory.server.DirectoryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.messaging.Message;
+import org.springframework.stereotype.Service;
+
+@Service
```
Comment (niconoir) [inline src/main/java/org/gridsuite/directory/server/services/ConsumerService.java:13]: Missing author

**Review by niconoir — COMMENTED** [review]
Just missing header

File: src/main/java/org/gridsuite/directory/server/services/ConsumerService.java (hunk 2 above)
Comment (archendel) [reply src/main/java/org/gridsuite/directory/server/services/ConsumerService.java:13]: fixed

File: src/test/java/org/gridsuite/directory/server/DirectoryTest.java [hunk 3]
```diff
@@ -99,6 +110,9 @@ public class DirectoryTest {
     @Autowired
     private OutputDestination output;
 
+    @Autowired
+    InputDestination input;
```
Comment (Meklo) [inline src/test/java/org/gridsuite/directory/server/DirectoryTest.java:114]: private ?

**Review by niconoir — APPROVED** [review]

File: src/test/java/org/gridsuite/directory/server/DirectoryTest.java (hunk 3 above)
Comment (archendel) [reply src/test/java/org/gridsuite/directory/server/DirectoryTest.java]: fixed

**Review by Meklo — APPROVED** [review]

**Pull request merged**